drghs-worker/kubernetes/deployment.yaml

```diff
@@ -193,4 +193,26 @@ metadata:
   name: drghs-endpoints-cert
 spec:
   domains:
-    - drghs.endpoints.PROJECT_ID.cloud.goog
\ No newline at end of file
+    - drghs.endpoints.PROJECT_ID.cloud.goog
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: hello
+spec:
+  # Run Every 2 Hours
```

Comment:
So this job will be using the access token of the `drghs` GitHub user, and there are 500+ repos for a total of 121k issues. How is this going to work with the limit 5000 API calls per hour on that access token?

Reply:
Based on my testing, each "list issues" call uses up 1 "point" of the available 5000 per hour

Reply:
Correct on rate limits. That said it should not start new jobs if an existing job is running.

I've been thinking about this more, and we _might_ be able to get better performance out of that by specifying multiple repositories in the query, but probably not all of them.

Reply:
If every ListIssues call retrieves 100 issues, we can retrieve up to 500,000 issues an hour, which is enough for now. To retrieve 121k issues it will take approximately 2100 API calls. This will work for now as long as the `drghs` token doesn't have it rate limit eaten up by another worker.

Perhaps, change this cron so it runs ~3 times a day for now and we'll see how it goes and if it needs to be more or less often?

Reply:
Thoughts?

Reply:
How do you feel about once a day?

Reply:
Once a day, let's do that